single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/AbstractVoltageStyleProvider.java

```diff
@@ -49,6 +49,9 @@ public List<String> getEdgeStyles(Graph graph, Edge edge) {
                 .map(Optional::get)
                 .collect(Collectors.toList());
         list.add(0, WIRE_STYLE_CLASS);
+        if (edge.isOverloaded()) {
+            list.add(StyleClassConstants.OVERLOAD_STYLE_CLASS);
+        }
```

Comment:
Can you clean the WIRE_STYLE_CLASS just above your change :
`list.add(0, WIRE_STYLE_CLASS);` => `list.add(0, StyleClassConstants.WIRE_STYLE_CLASS);`
and remove the static import line 24